python_files/generate_supplementary_plots.py

```diff
@@ -501,6 +501,77 @@
 )
 
 
-# Feature extraction
-
-
+# False positive analysis
+## Analyse the significance scores of top features after randomization compared to the TONIC data.
+fp_dir = os.path.join(SUPPLEMENTARY_FIG_DIR, 'false_positive_analysis')
+if not os.path.exists(fp_dir):
+    os.makedirs(fp_dir)
+
+# compute random feature sets
+'''
+combined_df = pd.read_csv(os.path.join(ANALYSIS_DIR, 'timepoint_combined_features.csv'))
+feature_df = pd.read_csv(os.path.join(ANALYSIS_DIR, 'feature_ranking.csv'))
+feature_metadata = pd.read_csv(os.path.join(ANALYSIS_DIR, 'feature_metadata.csv'))
+
+repeated_features, repeated_features_num, scores = [], [], []
+similarity_scores, random_top_features = [], []
+
+sample_num = 100
+np.random.seed(13)
+
+for seed in random.sample(range(1, 2000), sample_num):
+    _, jaccard_score, top_random_features = supplementary_plot_helpers.\
+        random_feature_generation(combined_df, seed, feature_df[:100])
+
+    df = pd.DataFrame({
+        'random_seed': [seed],
+        'jaccard_score': [jaccard_score]
+    })
+    supplementary_plot_helpers.append(df)
+
+    top_random_features['seed'] = seed
+    random_top_features.append(top_random_features)
+
+results = pd.concat(similarity_scores)
+top_features = pd.concat(random_top_features)
+# add TONIC features to data with seed 0
+top_features = pd.concat([top_features, feature_df[:100]])
+top_features['seed'] = top_features['seed'].fillna(0)
+
+#results.to_csv(os.path.join(fp_dir, 'similarity_scores.csv'), index=False)
+#top_features.to_csv(os.path.join(fp_dir, 'top_features.csv'), index=False)'''
```

Comment:
I think you had one or two plots looking at the specific shared features. Maybe it was like a jitter plot or something?

Reply:
Oops yeah forgot to include the original shared features plots. I'll add those in.